test/peripheral/521_borrowProxy_base.ts

```diff
@@ -198,8 +200,8 @@ contract('BorrowProxy', async (accounts) => {
             WETH,
             maturity1,
             user2,
+            oneToken.mul(2),
```

Comment:
as above
```suggestion
            oneToken,
```

Reply:
This one needs to stay, so that in the test starting on line 282 the user has more than one token in debt, and his repayment is taken whole.

The test starting on line 311 verifies that when trying to repay more than the existing debt, only the needed amount is taken.